src/Standards/BestIt/CodeSniffer/File.php

```diff
@@ -13,91 +15,67 @@
  * with int returns (false => -1)
  * Additionally there could be some architecture changes in the future, like Token-Objects and so on.
  *
- * @package BestIt\CodeSniffer
  * @author Nick Lubisch <nick.lubisch@bestit-online.de>
+ * @package BestIt\CodeSniffer
  */
-class File
+class File extends AbstractFileDecorator
```

Comment:
Created a "correct" decorator, even if I thing, this logic is unnecessary since the beginning. But now, to hard remove completely.